kvbc/test/v4blockchain/v4_blockchain_test.cpp

```diff
@@ -45,6 +48,106 @@ class v4_kvbc : public Test {
     ASSERT_EQ(0, db.use_count());
     cleanup();
   }
+  void create_blocks(uint64_t num_blocks,
```

Comment:
@arc-vmware thanks for the effort, this is great!

Reply:
Thanks @cloudnoize